### ISSUE TITLE
Cloud roles not accessing correct variables?

### ISSUE THREAD

**Issue by aboutte** [trailofbits/algo#450]
I just tried a EC2 deployment from master and hit the following exception:

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: <Response><Errors><Error><Code>AuthFailure</Code><Message>AWS was not able to validate the provided access credentials</Message></Error></Errors><RequestID>cb6cd4c9-12b4-482b-a8e9-e7ebdcdb0a2d</RequestID></Response>
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Traceback (most recent call last):\n  File \"/var/folders/9g/bkswtmgn1s15537xmyytb19m0000gn/T/ansible_2MHZFz/ansible_module_ec2_ami_find.py\", line 423, in <module>\n    main()\n  File \"/var/folders/9g/bkswtmgn1s15537xmyytb19m0000gn/T/ansible_2MHZFz/ansible_module_ec2_ami_find.py\", line 366, in main\n    images_result = ec2.get_all_images(owners=owner, filters=filter)\n  File \"/Users/aboutte/Documents/r/algo/env/lib/python2.7/site-packages/boto/ec2/connection.py\", line 189, in get_all_images\n    [('item', Image)], verb='POST')\n  File \"/Users/aboutte/Documents/r/algo/env/lib/python2.7/site-packages/boto/connection.py\", line 1186, in get_list\n    raise self.ResponseError(response.status, response.reason, body)\nboto.exception.EC2ResponseError: EC2ResponseError: 401 Unauthorized\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Response><Errors><Error><Code>AuthFailure</Code><Message>AWS was not able to validate the provided access credentials</Message></Error></Errors><RequestID>cb6cd4c9-12b4-482b-a8e9-e7ebdcdb0a2d</RequestID></Response>\n", "module_stdout": "", "msg": "MODULE FAILURE"}
```

TLDR exception

```
AWS was not able to validate the provided access credentials
```

It looks like this [commit](https://github.com/trailofbits/algo/commit/9c12272c8cc80f5dc0ff2fc073f88cc70f7a0a6c) changed all the ansible fact variables.

For example in ec2 the facts are:

```
aws_access_key
aws_secret_key
```

but the code tries to access the following variables:

```
access_key
secret_key
```

I am not an ansible guy but this looks like an issue to me.  Am I missing something?

**Comment by gunph1ld** [comment]
There are no bugs in the role and I can not reproduce it, works as expected.
The facts `access_key, secret_key` are coming from `aws_access_key and aws_secret_key` variables or envs

**Comment by gunph1ld** [comment]
@aboutte any updates?

**Comment by aboutte** [comment]
Not sure what I screwed up the other day but I can confirm things are working as expected now.
